### ISSUE TITLE
Command Requests = Compatibility, Games, Actions.

### ISSUE THREAD

**Issue by Korral** [avaire/avaire#16]
[version]: https://github.com/avaire/avaire/blob/master/build.gradle#L10
[guild]: https://discord.gg/gt2FWER
[stack overflow]: https://stackoverflow.com/questions/tagged/java

## General Troubleshooting

Hey there! Before you report a bug or suggest a new feature,
  please make sure to follow these steps first!
  
- [y] I have checked for similar issues.
- [y] I have updated to the [latest Ava version][version].
- [y] I have checked the branches or the maintainers' PRs for upcoming features/bug fixes.

> This is not the place to learn Java. Please refer to [StackOverflow][stack overflow]
  for your general programming questions.

### Issue Type

- [ ] Bug Report
- [Y] Feature Request

> For questions join the official AvaIre discord server: https://discord.gg/gt2FWER
[Y] means Yes

### Description
So basically compatilibility is like shipping. To test how well you and someone else you specify would do as a couple. [as a jooke.] Ranging from 0% to 100%.  It'd not only allow actual users in the server, but other names like ex. Derp [whos not in a server] x Derp 2 [also not in server]. Kinda like that. And games will entertain people. Such as Russian Roulette, playing with other friends, or even battles. And actions are gifs that show that same action. EX. someone slapping someone, showing a gif of someone slapping another person in anime, etc.

**Comment by Senither** [comment]
The actions you're talking about already exists, in Ava they're called [interactions](https://avairebot.com/docs/master/commands#interaction)
![slap](https://user-images.githubusercontent.com/8415349/41012667-fb9353a0-6942-11e8-9ce6-1d7fc9d47f85.png)
Interactions has a bunch of different gifs for each type that are chosen at random each time, everything else sounds pretty intersting so I'll most likely give it a go at some point if no one else wants to implement it ^^

**Comment by Kaguya-chan** [comment]
This thread has been made 10 months ago and since no one's working on it (and I quite like these features too), can you work on it Senither?
